diff -urNad freeipa-1.1.0~/ipa-server/ipa-kpasswd/Makefile.am freeipa-1.1.0/ipa-server/ipa-kpasswd/Makefile.am
--- freeipa-1.1.0~/ipa-server/ipa-kpasswd/Makefile.am	2008-06-11 16:24:28.000000000 +0200
+++ freeipa-1.1.0/ipa-server/ipa-kpasswd/Makefile.am	2008-07-09 07:43:50.000000000 +0200
@@ -43,7 +43,7 @@
 	*~			\
 	Makefile.in
 
-initdir=$(sysconfdir)/rc.d/init.d
+initdir=$(sysconfdir)/init.d
 
 install-data-hook: ipa_kpasswd.init
 
