Author: Timo Aaltonen <tjaalton@ubuntu.com>
Date:   Fri Oct 28 15:44:52 2011 +0300

    Add fix_dnsclient.py.patch, we don't use authconfig.

--- a/ipapython/dnsclient.py
+++ b/ipapython/dnsclient.py
@@ -18,7 +18,7 @@
 import socket
 import sys
 
-import acutil
+#import acutil
 
 DNS_C_IN = 1
 DNS_C_CS = 2
@@ -456,7 +456,7 @@
 	qdata = dnsFormatQuery(query, qclass, qtype)
 	if not qdata:
 		return []
-	answer = acutil.res_send(qdata)
+	answer = None
 	if not answer:
 		return []
 	return dnsParseResults(answer)
